react/features/base/util/interceptComponent.js

```diff
@@ -17,14 +28,36 @@ const _RULES = [
      * app even if the browser supports the app (e.g. Google Chrome with
      * WebRTC support on Android).
      *
+     * @param {Object} state - Redux state of the app.
+     *
      * @returns {UnsupportedMobileBrowser|void} If the rule is satisfied then
      * we should intercept existing component by UnsupportedMobileBrowser.
      */
     () => {
         const OS = Platform.OS;
 
         if (OS === 'android' || OS === 'ios') {
-            return UnsupportedMobileBrowser;
+            const mobileAppEnabled = interfaceConfig.MOBILE_APP_ENABLED;
```

Comment:
Yes, I removed some logic from here where this variable was used. So, let get rid of it.